repositories/dbmodels/db_case_event.go

```diff
@@ -5,19 +5,20 @@ import (
 
 	"github.com/checkmarble/marble-backend/models"
 	"github.com/checkmarble/marble-backend/utils"
+	"github.com/jackc/pgx/v5/pgtype"
 )
 
 type DBCaseEvent struct {
-	Id             string    `db:"id"`
-	CaseId         string    `db:"case_id"`
-	UserId         string    `db:"user_id"`
-	CreatedAt      time.Time `db:"created_at"`
-	EventType      string    `db:"event_type"`
-	AdditionalNote *string   `db:"additional_note"`
-	ResourceId     *string   `db:"resource_id"`
-	ResourceType   *string   `db:"resource_type"`
-	NewValue       *string   `db:"new_value"`
-	PreviousValue  *string   `db:"previous_value"`
+	Id             string      `db:"id"`
+	CaseId         string      `db:"case_id"`
+	UserId         pgtype.Text `db:"user_id"`
```

Comment:
If we do save nillable user id, it would be great to adapt the event DTO to return nillable values too (and not `''`)

Reply:
Of course, that makes sense. I pushed another commit